src/pages/LibraryPage.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <q-page class="column bg-grey-1">
+    <q-card class="no-border-radius">
+      <q-card-section class="q-my-none">
+        <q-btn
+          flat
+          dense
+          size="sm"
+          color="primary"
+          data-cy="page_library_go_back"
+          :label="$t('LibraryPage.text.goBack')"
+          :icon="$t('LibraryPage.icon.goBack')"
+          @click="$router.push('/libraries')"
+        />
+      </q-card-section>
+      <q-card-section class=" q-py-none">
+        <h4
+          class="q-ma-none"
+          data-cy="page_library_title"
+        >
+          {{ $t('LibraryPage.text.title', library) }}
+        </h4>
+      </q-card-section>
+      <q-card-section>
+        <div class="row">
+          <div class="q-mr-md">
+            <span class="q-mr-sm text-bold">{{ $t('LibraryPage.text.version') }}</span>
+            <span data-cy="page_library_version">{{ library.version }}</span>
+          </div>
+          <div>
+            <span class="q-mr-sm text-bold">{{ $t('LibraryPage.text.author') }}</span>
+            <span data-cy="page_library_author">{{ library.author }}</span>
+          </div>
+        </div>
+        <div>
+          <span class="q-mr-sm text-bold">{{ $t('LibraryPage.text.description') }}</span>
+          <span data-cy="page_library_description">{{ library.description }}</span>
+        </div>
+      </q-card-section>
+      <q-linear-progress
+        v-if="loading"
+        indeterminate
+        color="primary"
+      />
+    </q-card>
+  </q-page>
+</template>
+
+<script setup>
+import { onMounted, ref } from 'vue';
+import { useRoute, useRouter } from 'vue-router';
+import { getLibrary } from 'src/composables/LetoModelizerApi';
+import { Notify } from 'quasar';
+import { useI18n } from 'vue-i18n';
+
+const loading = ref(true);
+const { t } = useI18n();
+const route = useRoute();
+const router = useRouter();
+const library = ref({});
+
+/**
+ * Load library from id in url. If library not exists redirect to libraries page.
+ * @returns {Promise<void>} Return nothing on success.
```

Comment:
Promise with... otherwise...

Reply:
done

Reply:
otherwise...

Reply:
We will never have an error on it promise. So not to do.